app/tests/templates/workstation.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div>
+        Template: <b>{{ view.template_name }}</b>
+    </div>
+    <h1>Messages:</h1>
+    <ol id="messages"></ol>
+    <h1>Acks:</h1>
+    <ol id="acks"></ol>
+{% endblock %}
+
+{% block script %}
+    <script type="text/javascript">
+        function handleMessage(event) {
+            const eventString = JSON.stringify(event.data)
+            const node = document.createElement("li");
+            node.innerHTML = eventString;
+            const messages = document.getElementById("messages");
+            messages.appendChild(node);
+
+            const msg = {
+                sessionControl: {
+                    header: {

```

Comment:
Could we change this from `header` to `meta`?

Reply:
only heere or do you also want to change that in the postmessage that we send in session-control.js (where we send the id along)?

Reply:
Everywhere, as the default place to add meta items such as id and acks.